introduction.md

```diff
@@ -1 +1,4 @@
 Marc-san at your service. I prefer the path of HTML/CSS. Hai!
+* Sassy
+* Semantic markup
+* Git-fu
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?